src/robot_properties_solo/utils.py

```diff
@@ -0,0 +1,97 @@
+try:
+    # use standard Python importlib if available (>Python3.7)
+    import importlib.resources as importlib_resources
+except ImportError:
+    import importlib_resources
+
+import glob
+import sys
+from os import path, walk, mkdir, access, X_OK, environ, pathsep
+import subprocess
+
+from xacro import process_file, open_output
+from xacro.color import warning, error, message
+from xacro.xmlutils import *
+from xacro.cli import process_args
+
+try: # python 2
```

Comment:
Do we support python2 still? If not, I would remove the try-except block and just have the python3 code.